src/bp/migrations/v12_1_0-1564683507-remove_migrations_tables.ts

```diff
@@ -0,0 +1,21 @@
+import * as sdk from 'botpress/sdk'
+import { Migration } from 'core/services/migration'
+
+const migration: Migration = {
+  info: {
+    description: 'Remove migrations tables',
+    type: 'database'
+  },
+  up: async ({ bp }: sdk.ModuleMigrationOpts): Promise<sdk.MigrationResult> => {
```

Comment:
Not necessary, but could use the MigrationOpts not from the sdk (since it's the core)